src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -100,6 +100,16 @@ protected ActionResult execute() {
             data.emailToEdit.emailId = emailId;
         }
 
+        if (addressModeOn && groupModeOn) {
+            statusToAdmin = "Email queued for sending." + "<br/>" + "Receipient: " + addressReceiverListString
```

Comment:
Take a look at https://github.com/TEAMMATES/teammates/pull/7051/files#diff-fced2f82a76e423b56927175d900ca9aR70. 
"..group mode and address mode both are not on.." won't happen.